ecs/spotfleet-cluster/variables.tf

```diff
@@ -0,0 +1,64 @@
+variable "prefix" {
+  description = "A prefix used for naming resources."
+}
+
+variable "ami" {
```

Comment:
We need to try to standardise the variable names we use, to avoid confusion. Look here:
https://github.com/TeliaSoneraNorge/telia-terraform-modules/blob/spotfleet/ecs/cluster/variables.tf#L17-L20

E.g., `ami` should be `instance_ami`, `subnets` should be `subnet_ids` etc.